javascript/calcs/equip/objects/equip_buffs_list.js

```diff
@@ -573,7 +577,7 @@ equip_buff_list = [
+    { "id": 182, "name": "Bloody Action - Festival of Blood", "icon": "bloody_action", "category": "class", "class": ["black_massacre"], "mode": ["mult", "add", "add", "mult", "mult"], "options": ["20 Stacks", "19 Stacks", "18 Stacks", "17 Stacks", "16 Stacks", "15 Stacks", "14 Stacks", "13 Stacks", "12 Stacks", "11 Stacks", "10 Stacks", "9 Stacks", "8 Stacks", "7 Stacks", "6 Stacks", "5 Stacks", "4 Stacks", "3 Stacks", "2 Stacks", "1 Stack",], "stats": ["patk_mod", "crit", "as", "ms", "js"], "values": [[20, 3 * 20, 0.6 * 20, 0.6 * 20, 0.6 * 20], [19, 3 * 19, 0.6 * 19, 0.6 * 19, 0.6 * 19], [18, 3 * 18, 0.6 * 18, 0.6 * 18, 0.6 * 18], [17, 3 * 17, 0.6 * 17, 0.6 * 17, 0.6 * 17], [16, 3 * 16, 0.6 * 16, 0.6 * 16, 0.6 * 16], [15, 3 * 15, 0.6 * 15, 0.6 * 15, 0.6 * 15], [14, 3 * 14, 0.6 * 14, 0.6 * 14, 0.6 * 14], [13, 3 * 13, 0.6 * 13, 0.6 * 13, 0.6 * 13], [12, 3 * 12, 0.6 * 12, 0.6 * 12, 0.6 * 12], [11, 3 * 11, 0.6 * 11, 0.6 * 11, 0.6 * 11], [10, 3 * 10, 0.6 * 10, 0.6 * 10, 0.6 * 10], [9, 3 * 9, 0.6 * 9, 0.6 * 9, 0.6 * 9], [8, 3 * 8, 0.6 * 8, 0.6 * 8, 0.6 * 8], [7, 3 * 7, 0.6 * 7, 0.6 * 7, 0.6 * 7], [6, 3 * 6, 0.6 * 6, 0.6 * 6, 0.6 * 6], [5, 3 * 5, 0.6 * 5, 0.6 * 5, 0.6 * 5], [4, 3 * 4, 0.6 * 4, 0.6 * 4, 0.6 * 4], [3, 3 * 3, 0.6 * 3, 0.6 * 3, 0.6 * 3], [2, 3 * 2, 0.6 * 2, 0.6 * 2, 0.6 * 2], [1, 3 * 1, 0.6 * 1, 0.6 * 1, 0.6 * 1]], "pvp_values": [[0.5 * 20, 1.5 * 20, 0.3 * 20, 0.3 * 20, 0.3 * 20], [0.5 * 19, 1.5 * 19, 0.3 * 19, 0.3 * 19, 0.3 * 19], [0.5 * 18, 1.5 * 18, 0.3 * 18, 0.3 * 18, 0.3 * 18], [0.5 * 17, 1.5 * 17, 0.3 * 17, 0.3 * 17, 0.3 * 17], [0.5 * 16, 1.5 * 16, 0.3 * 16, 0.3 * 16, 0.3 * 16], [0.5 * 15, 1.5 * 15, 0.3 * 15, 0.3 * 15, 0.3 * 15], [0.5 * 14, 1.5 * 14, 0.3 * 14, 0.3 * 14, 0.3 * 14], [0.5 * 13, 1.5 * 13, 0.3 * 13, 0.3 * 13, 0.3 * 13], [0.5 * 12, 1.5 * 12, 0.3 * 12, 0.3 * 12, 0.3 * 12], [0.5 * 11, 1.5 * 11, 0.3 * 11, 0.3 * 11, 0.3 * 11], [0.5 * 10, 1.5 * 10, 0.3 * 10, 0.3 * 10, 0.3 * 10], [0.5 * 9, 1.5 * 9, 0.3 * 9, 0.3 * 9, 0.3 * 9], [0.5 * 8, 1.5 * 8, 0.3 * 8, 0.3 * 8, 0.3 * 8], [0.5 * 7, 1.5 * 7, 0.3 * 7, 0.3 * 7, 0.3 * 7], [0.5 * 6, 1.5 * 6, 0.3 * 6, 0.3 * 6, 0.3 * 6], [0.5 * 5, 1.5 * 5, 0.3 * 5, 0.3 * 5, 0.3 * 5], [0.5 * 4, 1.5 * 4, 0.3 * 4, 0.3 * 4, 0.3 * 4], [0.5 * 3, 1.5 * 3, 0.3 * 3, 0.3 * 3, 0.3 * 3], [0.5 * 2, 1.5 * 2, 0.3 * 2, 0.3 * 2, 0.3 * 2], [0.5 * 1, 1.5 * 1, 0.3 * 1, 0.3 * 1, 0.3 * 1]] },
```

Comment:
this is revert of a previously done change, please remove it (caused by not rebasing the branch)